metecho/api/models.py

```diff
@@ -1049,6 +1040,31 @@ def finalize_refresh_org(self, *, error=None, originating_user_id):
             )
             self.queue_delete(originating_user_id=originating_user_id)
 
+    def queue_reassign(self, *, new_user, originating_user_id):
+        from .jobs import user_reassign_job
+
+        self.currently_refreshing_org = True
+        self.deleted_at = None
```

Comment:
I think this change should maybe happen in `finalize_`, after it's been successfully re-assigned?

Reply:
Actually, maybe not. But this brings up a related issue -- the front end does not subscribe to WS events for deleted objects. So it won't receive this message about un-deletion... We can talk through the options there.

Reply:
What sort of options are you considering? The obvious one to me is to stay subscribed to deleted ScratchOrgs.

Reply:
Right, but this may involve an org that was soft-deleted prior to a page refresh, so the front end never knew about it in the first place. It seems a bit much for the client to subscribe to every org every time, deleted or not?

I was thinking a new event (object un-deleted or whatnot), either on a new list-specific channel (a la https://trello.com/c/1TS05t3A) or on the User channel or something.